### DISCUSSION TITLE
Is there a way to show envelope for last touched FX param when changed by Realearn?

### DISCUSSION THREAD

**Discussion by robotronald** [helgoboss/helgobox#1089] [Q&A] [answered]
Using Realearn to change FX parameters on a soft synth. When I use the action 'FX: Show/hide track/take envelope for last touched FX parameter' it opens the lane on the track containing the Realearn instance. Is there any way to get this action (or use another) to show the envelope on the track containing the FX that's being changed?

**Comment by helgoboss** [comment, marked as answer]
Are you maybe changing a parameter within ReaLearn as well? Otherwise I wouldn't know why it would choose the ReaLearn track.

**Reply by robotronald** [reply]
Thanks, yes that was it. I'm using autoload for focused VST parameters and clicking in the arrange window was unloading the preset so Realearn was the 'last touched'. 

Enabling 'Ignore FX chain keyboard shortcuts' allows me to us this command without needing to unfocus the FX window, works fine when I do that so all's good :)

**Reply by on3iro** [reply]
@robotronald could please give me a hint, where the setting you mentioned could be found?

**Reply by robotronald** [reply]
@on3iro It's in the Options menu on the FX chain window. When you change the setting it applies on all FX chains in the project.

I very rarely used keyboard shortcuts in plugins so this wasn't a problem but the biggest issue for me is that you can't use the delete, F2=rename, etc. keys in the FX window when this is enabled so you have to be careful you don't accidentally delete tracks when you're trying to delete FX, etc.

**Reply by on3iro** [reply]
thanks
